packages/grafana-runtime/src/config.ts

```diff
@@ -141,7 +141,7 @@ export class GrafanaBootConfig implements GrafanaConfig {
   rudderstackDataPlaneUrl: undefined;
   rudderstackSdkUrl: undefined;
   rudderstackConfigUrl: undefined;
-
+  usageInsightsEnabled =  true;
```

Comment:
I guess this is safe even for those cases when the feature is disabled in Grafana settings, isn't it? 
Or should I defined it `undefined` by default? 🤔